doc/overview/geostyler_overview.rst

```diff
@@ -0,0 +1,71 @@
+:Author: Jan Suleiman
+:Reviewer: Astrid Emde
+:Version: osgeolive14.0
+:License: Creative Commons Attribution 3.0 Unported (CC BY 3.0)
+
+
+@LOGO_geostyler@
+@OSGEO_KIND_geostyler@
+
+
+@NAME_geostyler@
+================================================================================
+
+@NAME_geostyler@ is an Open Source JavaScript library that enables users to style maps with the help of a Graphical User Interface.
+The goal of @NAME_geostyler@ is a universally applicable GUI that works with any given style format; may it be SLD, QML, OpenLayers, etc.
```

Comment:
The "may it be" phrase isn't quite right. You could go with one of these options:

...format; be it SLD, QML or OpenLayers.
...format; for example, SLD, QML, OpenLayers, etc.

Reply:
done in https://github.com/OSGeo/OSGeoLive-doc/pull/604/commits/79b892a57dc759bc650410399b965dbbf2bcdd69